projects/storefrontapp-e2e-cypress/cypress/e2e/regression/my-account/update-email.e2e.cy.ts

```diff
@@ -27,15 +27,12 @@ describe('My Account - Update Email', () => {
       });
     });
 
-    describe('Logged in user', { testIsolation: false }, () => {
```

Comment:
with testIsolation, we can now also remove
win.sessionStorage.clear() in before() hook